parquet-hadoop/1.7.0.wso2v1/pom.xml

```diff
@@ -0,0 +1,103 @@
+    <name>WSO2 Carbon Orbit - Parquet-hadoop</name>
+    <description>
+        This bundle will represent Parquet-hadoop
+    </description>
+    <url>http://wso2.org</url>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.parquet</groupId>
+            <artifactId>parquet-hadoop</artifactId>
+            <version>${parquet.hadoop.version}</version>
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <version>2.4.0</version>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.apache.parquet.*;version="${project.version}";-split-package:=merge-first,
+                        </Export-Package>
+                        <Private-Package/>
+                        <Import-Package>
+                            !org.apache.parquet.*,
+                            org.apache.hadoop.*;version="[2.7.0,2.8.0)";resolution:=optional,
+                            org.apache.commons.codec.binary.*; version="[1.5.0,2.0.0)";resolution:=optional,
+                            org.codehaus.jackson.*; version="[1.9.0,2.0.0)";resolution:=optional,
+                            parquet.org.apache.thrift.*; version="0.0.0";resolution:=optional
+                        </Import-Package>
+                        <!--<DynamicImport-Package>*</DynamicImport-Package>-->
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <parquet.hadoop.version>1.7.0</parquet.hadoop.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
```

Comment:
Specifying the snapshot repository in the distribution management do not add a purpose because WSO2 orbit bundles do not have a snapshot version. Therefore it is recommended to remove the snapshot repository.